data/brands/shop/supermarket.json

```diff
@@ -9077,8 +9138,11 @@
         "brand": "יינות ביתן",
         "brand:en": "Yeinot Bitan",
         "brand:he": "יינות ביתן",
+        "brand:wikidata": "Q6561327",
         "name": "יינות ביתן",
         "name:en": "Yeinot Bitan",
+        "name:en1": "Yenot Bitan",
```

Comment:
`name:langcode+number` isn't a good key, you means those 2 name is also common used? Maybe `alt_name` can help?

Reply:
Hi, I changed it to alt name, thanks!